### PR TITLE
Add new Google analytics code.

### PR DESCRIPTION
**Pull request by rvincentcoveo**

### CODE REVIEW

**Comment by malaporte** [comment]
Did you test this locally in some way?

**Comment by rvincentcoveo** [comment]
Yes I ran the blog on my local machine.

File: _layouts/base.html [hunk 1]
```diff
@@ -83,6 +83,10 @@
   <meta name="author" content="{{ page.author.name }}" />
 {% endif %}
 
+{% if site.inc.analytics.google %}
```
Comment (malaporte) [inline _layouts/base.html]: why this change, out of sheer curiosity?

Comment (rvincentcoveo) [reply _layouts/base.html]: Because before, GA was loaded from the footer but this script needs to be in the `head`.
So I had to remove it from `_scripts.html` and paste it directly in the <head>

Comment (malaporte) [reply _layouts/base.html]: OK. I'll assume you checked that the GA data was properly recorded, and merge your PR.
